code-studio/src/js/components/GridEditor.jsx

```diff
@@ -141,3 +139,9 @@ window.dashboard.GridEditor = module.exports = React.createClass({
     </div>);
   },
 });
+module.exports = GridEditor;
+
+// We put this on the dashboard namespace because GridEditor is used by _grid.html.haml
+// TODO _ exposed just a renderer?
+window.dashboard = window.dashboard || {};
+window.dashboard.GridEditor = GridEditor;
```

Comment:
For GridEditor, we could make this part of the levelbuilder.js output (which seems more correct), but then we will still need to stick this on a global if we want to access it from _grid.html.haml